options.rb

```diff
@@ -1,6 +1,10 @@
 def show_options
-  puts "\nPlease choose and option by entering a number:
-1 - List all books\n2 - List all people\n3 - Create a person
-4 - Create a book\n5 - Create a rental
-6 - List all rentals for a given person id\n7 - Exit"
+  puts "\nPlease choose an option by entering a number:
+1 - List all books
+2 - List all people
+3 - Create a person
+4 - Create a book
+5 - Create a rental
+6 - List all rentals for a given person ID
+7 - Exit"
```

Comment:
- [ **OPTIONAL**] Clicking on `7` to exit doesn't exit in the first place it restarts the app and we have to click on `7 ` again to exit the app. I think it would be a good idea to fix that for a better user experience.  :ok_hand:  😊